packages/react-composites/src/composites/ChatComposite/file-sharing/FileCard.tsx

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import { Icon, mergeStyles, ProgressIndicator, Stack, Text, useTheme } from '@fluentui/react';
+import { getFileTypeIconProps } from '@fluentui/react-file-type-icons';
+import React from 'react';
+
+/**
+ * @beta
+ */
+export interface FileCardProps {
+  /**
+   * File name.
+   */
+  fileName: string;
+  /**
+   * Extension of the file used for rendering the file icon.
+   */
+  fileExtension: string;
+  /**
+   * File upload/download progress percentage between 0 and 1.
+   * Not shown if undefined or 1.
+   */
+  progress?: number;
+  /**
+   * Icon to display for actions like download, upload, etc. on the right of file name.
+   */
+  actionIcon?: JSX.Element;
+  /**
+   * Function that runs when actionIcon is clicked
+   */
+  actionHandler?: () => void;
+}
+
+/**
+ * @beta
+ */
+export const FileCard = (props: FileCardProps): JSX.Element => {
+  const { fileName, fileExtension, progress, actionIcon } = props;
+  const theme = useTheme();
+
+  const progressIndicator = (): JSX.Element => (
```

Comment:
Can this just be inline where it's used and not define a new component with the same name?
```
{progress !== undefined && progress < 1 && <ProgressIndicator styles={progressStyles} percentComplete={progress} />
```